psi4/driver/driver.py

```diff
@@ -1139,12 +1141,12 @@ def optimize(name, **kwargs):
 
         n += 1
 
-    core.print_out('\tOptimizer: Did not converge!')
     if core.get_option('OPTKING', 'INTCOS_GENERATE_EXIT') == False:
         if core.get_option('OPTKING', 'KEEP_INTCOS') == False:
             core.opt_clean()
 
     optstash.restore()
+    raise ConvergenceError("""geometry optimization""", n - 1)
```

Comment:
Cool, as discussed I think this will alleviate the `grep 'beer' output.dat` problem.